app/models/business.rb

```diff
@@ -1,10 +1,24 @@
 class Business < ApplicationRecord
+	# Todo update method name to reflect search criteria
+	# Todo Add weights to columns
+	# Additonal options => normalization, sort_only, any_word
+	#* Default results are ordered by relevance
+	include PgSearch::Model
+	pg_search_scope :search_by_theme,
```

Comment:
Are you still using the scope strategy?

Reply:
I ended up sticking with it for now. I made a branch for the multi-search, and when comparing the two the scope strategy seemed to get the job done. I want to circle back though and integrate [tsvector columns](https://github.com/Casecommons/pg_search#using-tsvector-columns) for some speed 🐎